src/pages/index.tsx

```diff
@@ -1,136 +1,39 @@
           </svg>
         </h1>
-        {!menuDrawerOpen && !satellitesDrawerOpen && !dateDrawerOpen && (
-          <Button
-            className="ml-auto"
-            icon={<MdFormatListBulleted className="h-5 w-5" />}
-            onClick={() => {
-              setMenuDrawerOpen(true);
-              setDateDrawerOpen(false);
-              setSatellitesDrawerOpen(false);
-            }}
-            variant="secondary"
-          >
-            Menu
-          </Button>
-        )}
-
-        {(dateDrawerOpen || satellitesDrawerOpen || menuDrawerOpen) && (
-          <Button
-            className={clsx(
-              (dateDrawerOpen || satellitesDrawerOpen) && "ml-auto"
-            )}
-            icon={<MdClose className="h-5 w-5" />}
-            onClick={() => {
-              setDateDrawerOpen(false);
-              setSatellitesDrawerOpen(false);
-              setMenuDrawerOpen(false);
-            }}
-            variant="secondary"
-          >
-            Close
-          </Button>
-        )}
-      </Navbar>
-      <main className="relative h-screen overflow-y-hidden" ref={mainRef}>
-        <Scene
-          camera={{
-            far: 10000,
-            near: 0.0001,
-            position: [0, 0, 10],
-          }}
-        >
-          <Space />
-        </Scene>
-        <Progress />
-      </main>
-      <DateDrawer open={dateDrawerOpen} />
-      <SatellitesDrawer open={satellitesDrawerOpen} />
-      <MenuDrawer
-        open={menuDrawerOpen}
-        setDateDrawerOpen={setDateDrawerOpen}
-        setMenuDrawerOpen={setMenuDrawerOpen}
-        setSatellitesDrawerOpen={setSatellitesDrawerOpen}
-      />
-    </>
+        <Button className="">
+          <Link href="/space"> Launch SpaceTag</Link>
+        </Button>
+        <Link href="/space">
+          <h1 className=" text-xl text-slate-400 underline underline-offset-4">
+            {" "}
+            Learn More{" "}
+          </h1>
+        </Link>
```

Comment:
It is better to nest `<a>` inside of `<h1>` than the other way around.

```suggestion
        <h1 className=" text-xl text-slate-400 underline underline-offset-4">
          <Link href="/space">
            Learn More
          </Link>
        </h1>
```